test/ServerComparison.FunctionalTests/NtlmAuthenticationTest.cs

```diff
@@ -20,39 +20,29 @@ public NtlmAuthenticationTests(ITestOutputHelper output) : base(output)
         {
         }
 
+        public static TestList TestVariants
+            => TestMatrix.ForServers(ServerType.IISExpress, ServerType.HttpSys)
+                .WithTfms(Tfm.NetCoreApp22, Tfm.Net461)
+                .WithAllAncmVersions()
+                .WithAllHostingModels()
+                .Build();
+
         [ConditionalTheory]
         [OSSkipCondition(OperatingSystems.Linux | OperatingSystems.MacOSX)]
```

Comment:
Why do we still need skip condition here? Matrix should return no tests.

Reply:
A Theory with no results is an xunit error condition.